api/client/commands.go

```diff
@@ -49,11 +49,6 @@ func (cli *DockerCli) Command(name string) func(...string) error {
 		"unpause":            cli.CmdUnpause,
 		"update":             cli.CmdUpdate,
 		"version":            cli.CmdVersion,
-		"volume":             cli.CmdVolume,
```

Comment:
This PR only moves the `volume` command (and subcommands under it) to cobra. After this PR we can continue to move other commands over to the new system. I wanted to keep the PR smaller.